src/integrationtest/java/com.microsoft.aad.msal4j/NationalCloudIT.java

```diff
@@ -75,7 +75,6 @@ private void assertAcquireTokenCommon(NationalCloud cloud) throws Exception{
 
         Assert.assertNotNull(result);
         Assert.assertNotNull(result.accessToken());
-        Assert.assertNotNull(result.refreshToken());
```

Comment:
>Assert.assertNotNull(result.refreshToken()); [](start = 8, length = 44)

Why remove this check?

Reply:
we do not expose RT through IAuthenticationResult